### PR TITLE
coprocessor: add ddl schema version in request

### PR DESCRIPTION
**Pull request by lzmhhh123**
### What problem does this PR solve? 
When DDL has changed. TiFlash needs to know the DDL change to decode the correct data. So for TiFlash, it hopes TiDB lets it know the newest schema version in coprocessor request.

### What is changed and how it works?
Set the `SchemaVer` for `coprocessor.Request`.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity

### CODE REVIEW

File: executor/table_reader.go [hunk 1]
```diff
@@ -215,6 +219,7 @@ func (e *TableReaderExecutor) buildResp(ctx context.Context, ranges []*ranger.Ra
 		SetFromSessionVars(e.ctx.GetSessionVars()).
 		SetMemTracker(e.memTracker).
 		SetStoreType(e.storeType).
+		SetSchemaVer(txn).
```
Comment (tiancaiamao) [inline executor/table_reader.go]: `SchemaVer` is stored in the SessionVars.TxnCtx.SchemaVersion.
https://github.com/pingcap/tidb/blob/master/session/session.go#L1945
 @lzmhhh123

**Comment by tiancaiamao** [comment]
LGTM

File: distsql/request_builder.go [hunk 2]
```diff
@@ -165,6 +165,7 @@ func (builder *RequestBuilder) SetFromSessionVars(sv *variable.SessionVars) *Req
 	builder.Request.NotFillCache = sv.StmtCtx.NotFillCache
 	builder.Request.Priority = builder.getKVPriority(sv)
 	builder.Request.ReplicaRead = sv.GetReplicaRead()
+	builder.Request.SchemaVar = sv.TxnCtx.SchemaVersion
```
Comment (wjhuang2016) [inline distsql/request_builder.go]: Please update the comment of this function.

Comment (wjhuang2016) [reply distsql/request_builder.go]: Emmm, please update `Priority` in comment too.

**Pull request merged**